util/codec/codec.go

```diff
@@ -205,15 +206,11 @@ func encodeChunkRow(sc *stmtctx.StatementContext, b []byte, row chunk.Row, allTy
 				break
 			}
 			// encode unsigned integers.
-			if hash {
```

Comment:
`hash` is always true in the codebase, so I remove the other branch to improve test coverage.